### DISCUSSION TITLE
Hide/Delete Comments has disappeared from Comment Options

### DISCUSSION THREAD

**Discussion by Dsotm444** [community/community#36653] [Projects and Issues]
Hi

When in an issue comment, both the Hide and Delete options are no longer showing in the Comment Actions Dropdown menu. Is there a setting somewhere to turn this on? I am admin on all repositories and this used to be there before?

![image](https://user-images.githubusercontent.com/97218233/196695830-d7d4dedc-ad8e-48db-b7eb-03e83126d642.png)

**Comment by ossanna16** [comment]
Hey there @Dsotm444 and thank you for asking a good question! I'm glad you found our community 🙂
